### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Shevtsooov**
[DEMO LINK](https://Shevtsooov.github.io/react_tabs/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,42 +1,30 @@
...
 
-export const tabs = [
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0]);
```
Comment (silchyn) [inline src/App.tsx:15]: But `tab[0]` is not id

File: src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -1 +1,47 @@
...
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (tab.id !== selectedTabId) {
+                    onTabSelected(tab);
+                  }
+                }}
```
Comment (silchyn) [inline src/components/Tabs/Tabs.tsx:34]: Put it in a separate callback

**Review by silchyn — CHANGES_REQUESTED** [review]

File: src/components/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -1 +1,51 @@
...
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleClick = (tab: Tab) => {
```
Comment (33Orange) [inline src/components/Tabs/Tabs.tsx:17]: This function is not a `handler` for click

```suggestion
  const selectTab = (tab: Tab) => {
```

**Review by 33Orange — APPROVED** [review]
Good job

**Pull request closed without merging**